test/integration/default/inspec.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: cloud-pubsub
+title: Google Cloud Pub/Sub
+version: 0.1.0
+inspec_version: '~> 2.3.24'
+depends:
+  - name: inspec-terraform
```

Comment:
This dependency doesn't appear to be used in this module, and kitchen-terraform's `terraform` verifier is better suited to transferring Terraform outputs into InSpec as attributes.

Reply:
Oops, removed: https://github.com/terraform-google-modules/terraform-google-pubsub/pull/1/commits/51b9a5dc130e4d6287cf1bcaae5b09020f386fd6